### PR TITLE
Updated README with links to v3

### PR DESCRIPTION
**Pull request by droyad**
Related to Nuget/Home#3287

### CODE REVIEW

**Comment by droyad** [comment]
Related to Nuget/Home#3287

**Comment by yishaigalatzer** [comment]
LGTM, we should probably also update the description of nuget.core along the lines of what the original issue suggets. @harikmenon can you suggest the updates (both here and on nuget.org)

**Comment by harikmenon** [comment]
Adding @karann-msft : In our new nuget site, we have a page dedicated to talking about V3. http://nugetdocsbeta.azurewebsites.net/ndocs/api/nuget-api-v3. We should probably point to that instead of the just the first post by Dave. 

@karann-msft can you work with @droyad to make this change to the readme and get the contribution in?

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # NuGet2
-This is the home of nuget.core and all the repos from codeplex.
+This is the home of nuget.core and all the repos from codeplex. This repository is for Version 2 of NuGet. Version 3 of the Nuget client library has moved to [Nuget.Client](https://github.com/NuGet/nuget.client/). See [this blog post](http://daveaglick.com/posts/exploring-the-nuget-v3-libraries-part-1) to get an overview of the changes between version 2 and 3.
```
Comment (karann-msft) [inline README.md]: @droyad Please make the change to point to this URL - http://nugetdocsbeta.azurewebsites.net/ndocs/api/nuget-api-v3

Comment (droyad) [inline README.md]: Done

**Pull request merged**